lib/sass/script/functions.rb

```diff
@@ -1670,6 +1687,42 @@ def feature_exists(feature)
     end
     declare :feature_exists, [:feature]
 
+    # Returns a reference to a function for later invocation with the `call` function.
+    #
+    # The function reference created may refer to a function defined in
+    # your stylesheet, built-in to the host environment, or a CSS native
+    # function. As such, this function never returns an error, but the
+    # you can use `function-exists()` to check if the function reference
+    # created points at a Sass function.
+    #
+    # @example
+    #   function-reference(rgb)
```

Comment:
I think we should encourage users to use quotes here—both because I think strings should generally be quoted when they aren't specifically going to be emitted in CSS, and because we specifically have built-in functions with names like `red()` where unquoted names will break. Unquoted names also won't work in the module world with namespaced functions.